docs/firefly/verify_download.md

```diff
@@ -45,22 +46,22 @@ certUtil -hashfile C:\Users\yourname\Downloads\firefly-desktop-1.0.0.exe SHA256
 #### For Firefly 1.2.0 and earlier
 3. In the Certification Path tab, make sure that the path matches the following information:
 
-    - DigiCert
-    - DigiCert SHA2 Assured Code Signing CA
-    - IOTA Stiftung
+    - DigiCert.
+    - DigiCert SHA2 Assured Code Signing CA.
+    - IOTA Stiftung.
 
-4. Make sure that the Certificate status reads "This certificate is OK.".
+4. Make sure that the Certificate status reads, "This certificate is OK.".
```

Comment:
I don't think that this comma belongs here.

Reply:
Quotes preceded by verbs get commas. "Signal verbs". Depends on your grammar style so ymmv.

Reply:
Is it actually a quote? :thinking: 

> The plate says "painted".
> The plate says, "painted".
> The plate says: "painted".

@lucas-tortora wdyt?

Reply:
Gang up here, but removed the comma 😒